elfi/visualization/visualization.py

```diff
@@ -277,3 +277,68 @@ def progress_bar(iteration, total, prefix='', suffix='', decimals=1, length=100,
     print('\r%s |%s| %s%% %s' % (prefix, bar, percent, suffix), end='\r')
     if iteration == total:
         print()
+
+
+def plot_some(node, n_samples=100, seed=None, axes=None, **kwargs):
```

Comment:
Should it has more descriptive name?

Reply:
Suggestions?

The idea was to investigate summary vs. parameter behavior by plotting some node values i.e. `plot_some(node)`. :)

Reply:
Should probably start with `plot_`. So you don't like *some*? :)

Reply:
I **can** live with _some_, but maybe sth more descriptive? plot_explored(node)? plot_simulated(node)? :)